src/RulesTest/Library/NameIsEmptyButElementIsNotKeyboardFocusableTest.cs

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
```

Comment:
In "using System;" added intentionally? No changes in this file seem to necessitate it